src/Tools/dotnet-user-jwts/src/Helpers/DevJwtCliHelpers.cs

```diff
@@ -125,26 +126,66 @@ public static string[] GetAudienceCandidatesFromLaunchSettings(string project)
                     var profilesEnumerator = profiles.EnumerateObject();
                     foreach (var profile in profilesEnumerator)
                     {
-                        if (profile.Value.TryGetProperty("commandName", out var commandName))
+                        if (ExtractKestrelUrlsFromProfile(profile) is { } kestrelUrls)
                         {
-                            if (commandName.ValueEquals("Project"))
-                            {
-                                if (profile.Value.TryGetProperty("applicationUrl", out var applicationUrl))
-                                {
-                                    var value = applicationUrl.GetString();
-                                    if (value is { } applicationUrls)
-                                    {
-                                        return applicationUrls.Split(';');
-                                    }
-                                }
-                            }
+                            applicationUrls.AddRange(kestrelUrls);
+                        }
+
```

Comment:
nit: else if

Reply:
I don't think that's the right call here. We want to populate audience from both IIS and Kestrel-based URLs here so we get a complete set of audiences by the end.

Reply:
The reason I think it should be else if is because if the Kestrel check of `profile.Value.TryGetProperty("commandName", out var commandName)` passes then the IIS check won't pass for this specific `profile`. Again, it's a nit, not code we need to worry about performance for.